wpilibc/src/main/native/include/frc/buttons/POVButton.h

```diff
@@ -13,7 +13,7 @@
 namespace frc {
 class POVButton : public Button {
  public:
-  POVButton(GenericHID &joystick, int angle, int povNumber);
+  POVButton(GenericHID& joystick, int angle, int povNumber);
   virtual ~POVButton() = default;
 
   POVButton(POVButton&&);
```

Comment:
This should be `POVButton(POVButton&&) = default;` followed by `POVButton& operator=(POVButton&&) = default;` to enable move semantics.